src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -303,7 +305,11 @@ TestLocalizationConstant = test
 ############################################################################
 # Utilities
 
-SqlScriptFormatterDecimalMissingPrecision = Decimal column is missing numeric precision or numeric scale
+SqlScriptFormatterDecimalMissingPrecision = Exact numeric column is missing numeric precision or numeric scale
```

Comment:
Minor: if you can make sure these errors align to SSMS / SSDT errors, that'd be great.

Reply:
E.g. if we already have standardized string error messages it's nice to reuse as they were previously reviewed & localized.